### ISSUE TITLE
[Question] Query database output

### ISSUE THREAD

**Issue by zaee-k** [coleifer/peewee#2076]
Hey there!
First of all a big big thank you for creating peewee, it's a nice module to work with PostgreSQL and SQLite!
I have one silly question in regard of querying database. Currently I'm playing with peewee and FastAPI. One thing I've noticed is upon executing select(), data is of course queried and returned BUT together with some metadata like "__data__", "_dirty" and "__rel__". 
How can I get only the data which is stored in db EXCEPT that metadata? I might of course just read "__data__" value from the output, but when using aliases, my alias and value is outside "__data__". Here example to visualize:

What I get now:
```
[
   {
      "__data__":{
         "id":25052,
         "l":"abc",
         "asd":"bbd",
         "ip":"218.1.1.1",
         "timestamp":"2019-12-17T11:57:46.790299"
      },
      "_dirty":[

      ],
      "__rel__":{

      }
   }
]
```

What I want to get:

```
[
   {
      "id":25052,
      "l":"abc",
      "zxc":"def",
      "ip":"218.1.1.1",
      "timestamp":"2019-12-17T11:57:46.790299"
   }
]
```

TL;DR: How to hide "_dirty", "_rel" and remove "__data__" as key for actual data?

**Comment by coleifer** [comment]
>What I get now:

That is *not* returned by executing a `select()` query. That is metadata on the model instance. By default, issuing a `select()` will return a cursor that yields model instances for each row of the result set.

My guess is that you want something like `.dicts()`, which should give you exactly what you want. Just modify your query:

```python

query = MyModel.select()
for obj in query:
    # obj is a MyModel instance.
    ...

# This is probably what you want:
for row in query.dicts():
    # row is a dict of just the field data.
    ...
```

http://docs.peewee-orm.com/en/latest/peewee/querying.html#retrieving-row-tuples-dictionaries-namedtuples

**Comment by zaee-k** [comment]
Yes! That was the thing I've thought to do. All is good now, thanks! :)
